packages/highlight/src/lib/themes/monokaiPro/monokaiPro.ts

```diff
@@ -0,0 +1,65 @@
+import {defineEditorTheme} from '../../core';
+
+export const palette = {
+  constant: '#FFEE80',
+  foreground: '#2d2a2e',
+  numbers: '#AE81FF',
+  strings: '#E6DB74',
+  className: ' #59e366',
+  lineNumbers: '#535059',
+  punctuation: '#E1EFFF',
+  function: '#7be8b2',
+  keywords: '#5acfed',
+  cursor: '#f8f8f0',
+  comments: '#535059',
+  base: '#ffffff',
+  tag: '#9effff',
+  property: '#FFEE80',
+  selection: '#878b9180',
+  text: '#FAD000',
+  white: '#FFFFFF',
+  self:'#afb8ba',
+  module: '#d358db',
+  operator: '#fa0089',
+  annotation: '#fa0089'
+};
+
+export const monokaiPro = [
+  defineEditorTheme({
+    highlight: {
+      punctuation: palette.punctuation,
+      delimiters: palette.punctuation,
+      numbers: palette.numbers,
+      strings: palette.strings,
+      regexp: palette.strings,
+      variableName: palette.property,
+      keywords: palette.keywords,
+      base: palette.base,
+      tag: palette.tag,
+      boolean: palette.numbers,
+      comments: palette.comments,
+      propertyName: palette.property,
+      function: palette.function,
+      className: palette.className,
+      self: palette.self,
+      operators: palette.operator,
+      annotation: palette.annotation
+    },
+    selection: {
+      backgroundColor: palette.selection,
+      color: palette.white,
+    },
+    lineNumbers: {
+      color: palette.lineNumbers,
+    },
+    cursor: {
+      color: palette.cursor,
+    },
+    autocomplete: {
+      background: palette.strings,
+      border: palette.property,
+      selectedBackground: palette.strings,
+    },
+    darkMode: true,
+  }),
+];
```

Comment:
Colors doesn't look right in my opinion 🤔 . Monokai Pro as the site shows has 6 main colors:

```ts
{
  magenta: '#ff6188',
  cyan: '#78dce8',
  green: '#a9dc76',
  yellow: '#ffd866',
  orange: '#fc9867',
  indigo: '#ab9df2',
}
```

Some of the others colors are redundant, like the current green, cyan and the scale of gray-white


Here what I think we should change here:
- operator color should be yellow
- propertyColor should be green
- className should be green
- lineNumbers should be gray
- puntuaction should be gray
- keywords should be cyan

- definition, control and operators keyboard should be magenta
- `this` should be italic
- function variable call should be green (with the right hex)
- function variabled definition should be green


```ts
const colors = {
  magenta: '#ff6188',
  cyan: '#78dce8',
  green: '#a9dc76',
  yellow: '#ffd866',
  orange: '#fc9867',
  indigo: '#ab9df2',
  gray: '#939293FF',
  white: '#ffffff',
};

syntaxHighlighting(
    HighlightStyle.define([
      {
        tag: [t.definitionKeyword, t.controlKeyword, t.operatorKeyword],
        color: colors.magenta,
      },
      {tag: t.self, fontStyle: 'italic'},
      {tag: t.modifier, color: colors.cyan},
      {
        tag: t.function(t.variableName),
        color: colors.green,
      },
      {
        tag: t.definition(t.function(t.variableName)),
        color: colors.green,
      },
    ]),
 ),
```

![image](https://user-images.githubusercontent.com/37072694/194748665-b173eefd-ab2c-4664-ba1d-1fd368e29c1c.png)

Reply:
Sure, thanks will update them